include/rdm/responder.h

```diff
@@ -68,10 +68,20 @@
     (nack_resp)->command_class =                                                                          \
         ((cmd)->command_class == kRdmCCSetCommand ? kRdmCCSetCommandResponse : kRdmCCGetCommandResponse); \
     (nack_resp)->param_id = (cmd)->param_id;                                                              \
-    (nack_resp)->datalen = 2;                                                                             \
-    etcpal_pack_16b((nack_resp)->data, nack_reason);                                                        \
+    (nack_resp)->parameter_data.datalen = 2;                                                                             \
+    etcpal_pack_16b((nack_resp)->parameter_data.data, nack_reason);                                                        \
   } while (0)
 
+/*! \brief Parameter support flag definitions.
+ */
+#define RDM_PS_ROOT 0x00000001
+#define RDM_PS_SUBS 0x00000002
+#define RDM_PS_ALL (RDM_PS_ROOT | RDM_PS_SUBS)
+#define RDM_PS_SHOWSUP 0x80000000
```

Comment:
Remind me what `PS_SHOWSUP` means?

Reply:
Whether or not a PID should appear in the supported parameters list.

Reply:
Perhaps we should rename that one to RDM_PS_SHOW_SUPPORTED?

Reply:
Or negate the logic and call it `RDM_PS_HIDDEN`

Reply:
The hidden case is quite a bit less common than the show supported case, so I think that's a great idea. @nwagner your thoughts?

Reply:
In an offline discussion with Nick, he made a good point that if we invert the meaning of the flag, then existing users of the old library will have to be extra careful to account for this when adopting the new library. We settled on renaming the flag to RDM_PS_SHOW_SUPPORTED for now.